pkg/webui/styles/utilities/general.styl

```diff
@@ -367,43 +367,55 @@
 
   // DISPLAY UTILITIES.
   .d-flex
-    display: flex
+    // stylelint-disable-next-line declaration-no-important
```

Comment:
Please add one global ignore comment at the beginning and end of these definitions:
```
// stylelint-disable declaration-no-important
// stylelint-enable declaration-no-important
```